solr/src/main/scala/akka/stream/alpakka/solr/scaladsl/SolrFlow.scala

```diff
@@ -140,7 +140,28 @@ object SolrFlow {
           collection,
           client,
           settings,
-          binder
+          Some(binder)
+        )
+      )
+
+  /**
+   * Do not stream element to Solr. Just pass the pass-through data.
+   *
+   * @tparam PT pass-through type
+   */
+  def passThrough[T, PT](
```

Comment:
I do not understand the use-case for this flow.

Reply:
Yes you're right. I correct it.